    <launch>
        <node pkg="evaluater" type="evaluater" name="evaluater" output="screen">

                <param name="file_loc" value="/home/nuc/my/log/gpslam_3d_result/evaluater"/>

                <param name="lidar_install_x"            value="0.0"/>
                <param name="lidar_install_y"            value="0.0"/>
                <param name="lidar_install_z"            value="0.0"/>
                <param name="lidar_install_roll_degree"  value="0.0"/>
                <param name="lidar_install_pitch_degree" value="0.0"/>
                <param name="lidar_install_yaw_degree"   value="0.0"/>

                <param name="keyframe_delta_t" value="0.15"/>
                <param name="keyframe_delta_r" value="0.5"/>
                <param name="look_back"        value="3"/>
                <param name="window_size"      value="3"/>
                <remap from="/pose" to="/vrpn_client_node/RigidBody005/pose"/>
                <remap from="/velodyne_points" to="/turned_velodyned"/>
        </node>
    </launch>
